### PR TITLE
fix: make instance fns passable

### PR DESCRIPTION
**Pull request by chadoh**
This now works:

```js
async function testSomething(run) {
  await run(async ({ … }) => {
    // test code goes here
  })
}

const runner = await Runner.create(async ({ runtime }) => { … })

await testSomething(runner.run)
```

Previously, the `run` function in the context of `testSomething` would not have `this` defined, and confusing errors would be thrown.

### CODE REVIEW

File: tests/status-message/__tests__/main.spec.ts [hunk 1]
```diff
@@ -1,6 +1,13 @@
-import { Runner } from "../../../src";
+import { Runner, RunnerFn, Runtime } from "../../../src";
```
Comment (chadoh) [inline tests/status-message/__tests__/main.spec.ts:1]: this and everything below are goofy enough that I think we should create a separate `tests/*` directory called `edge-cases` or something, and move this there, or just write that test in JS instead of TS. But I don't think that's worth doing now.

**Comment by chadoh** [comment]
We can come back to this if people someone complains about this broken edge case. I don't want to complicate our main test file, which is currently used as much for teaching good practices as it is for actually testing. This way of testing is not a good practice! 

So until someone complains, our recommendation is to pass the whole `runner` around, rather than just `runner.run`.
